.circleci/config.yml

```diff
@@ -48,7 +48,21 @@ jobs:
       - run: CI=true npm run release:prep
       - run: TAG=$CIRCLE_TAG npm run release:validate
       - run: npm run publish
-
+  publish_example_server_container:
+    docker:
+      # image for building docker containers
+      - image: docker:17.05.0-ce-git
+    steps:
+      - checkout
+      # special workaround to allow running docker in docker https://circleci.com/docs/2.0/building-docker-images/
+      - setup_remote_docker:
+          version: 17.05.0-ce
+          docker_layer_caching: true
+      - run: |
+          cd server
+          [ -z "$DOCKERHUB_USERNAME" ] && echo "Undefined DOCKERHUB_USERNAME, skipping publish" && exit 0;
+          docker login -u $DOCKERHUB_USERNAME -p $DOCKERHUB_PASSWORD
+          TAG=$CIRCLE_TAG ./push-script.sh
```

Comment:
Maybe we can have everything above inside this script?

Reply:
moved